ArduPlane/tailsitter.cpp

```diff
@@ -114,13 +114,21 @@ void QuadPlane::tailsitter_output(void)
     // handle VTOL modes
```

Comment:
This comment should probably move down below the assisted flight code.

Reply:
fixed